### ISSUE TITLE
My output stream is empty.

### ISSUE THREAD

**Issue by daihz** [seisbench/seisbench#145]
Hi, my data is download from Japanese NEID KiK-net and K-NET. When I used model.annotate( ), the output stream is empty. I would like to know if my data lack of some informations within the stream. And when model.annotate( ) worked, the input stream should have what informations or conditions?

> import seisbench.models as sbm
import torch
from obspy.core import read
gpd_model = sbm.GPD.from_pretrained("instance")
stream = read('AIC0012010190328.EW')
stream += read('AIC0012010190328.NS')
stream += read('AIC0012010190328.UD')
preds = gpd_model.annotate(stream)
print(preds[0])

My obspy version is 1.3.1. Here is my data.
_https://drive.google.com/file/d/1srNTjMBGzZohrG0uYUZ9e0E30Or0jxm7/view?usp=share_link_
![image](https://user-images.githubusercontent.com/111106962/203244067-56feddfb-a8fb-46ec-bf32-36eefe6f218b.png)
![image](https://user-images.githubusercontent.com/111106962/203246504-ca9f33e1-0e1f-4e44-8405-1a9633a2db1d.png)

**Comment by yetinam** [comment]
Hey @daihz ,

thanks for the question. The issue results from the component naming of KiK-Net data. SeisBench expects standard [seed channel naming](https://ds.iris.edu/ds/nodes/dmc/data/formats/seed-channel-naming/), at least for the component identifier, i.e., the last letter. The components in KiK-Net are however called EW, NS and UD. Just rename them to E, N and Z, by overwriting `trace.stats.channel`. Make sure the channel codes are 3 letters. For KiK-Net, it should probably be `HN?` with the `?` indicating the component. The model should correctly pick up the traces then.

Feel free to reopen the issue if this does not solve the problem yet.
